Cargo.toml

```diff
@@ -14,7 +14,13 @@ anyhow = "1.0"
 clap = { version = "3.1", features = ["derive"] }
 fuel-crypto = "0.4"
```

Comment:
Should we also use the latest `fuel-crypto (v0.5)` and `fuel-types (v0.5.1)` (I guess they wouldn't break as they seem to be used in the latest version of [`fuels-rs`](https://github.com/FuelLabs/fuels-rs/blob/bca50740445a4dc3f09386cd2ed546de06f12ee0/packages/fuels-core/Cargo.toml#L15)).

Reply:
Definitely! good catch